build/tools/Automation/Services/YamlFileReaderService.cs

```diff
@@ -0,0 +1,27 @@
+// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.Oryx.Automation.Models;
+using YamlDotNet.Serialization;
+using YamlDotNet.Serialization.NamingConventions;
+
+namespace Microsoft.Oryx.Automation.Services
+{
+    public class YamlFileReaderService : IYamlFileReaderService
+    {
+        public async Task<List<YamlConstants>> ReadConstantsYamlFileAsync(string filePath)
+        {
+            string fileContents = await File.ReadAllTextAsync(filePath);
```

Comment:
We should have some error handling when dealing with I/O

Reply:
Updated